source/tutorials/index.rst

```diff
@@ -16,6 +16,7 @@ Get started with Valohai
     valohai/advanced/index
     cloud-storage/index
     code-repository/index
+    migrating_existing_projects
```

Comment:
We've used dashes instead of underscores previously, maybe this should follow suit?